devfiles/src/classes/Panel.ts

```diff
@@ -162,14 +162,14 @@ export default class Panel {
     public refresh(): void {
         //Update dataFilterers
         this.dataFilterer.dataUpdated();
-        this.refreshWidgets();
 
         //Update options to reflect new filters
         this.updateInputOptions();
         this.pageManager.refreshPanelOptions();
 
         //Refresh after internal class state is updated
         this.refreshComponent();
+        this.refreshWidgets();
```

Comment:
@SatoruQ You had moved it up, would moving it back down cause any problems for your code?

Reply:
Apparently not, I tested his code and it was ok